go/example/main.go

```diff
@@ -35,6 +35,23 @@ func printAllProjects(sdk *v4.LookerSDK) {
 	}
 }
 
+func printAboutMe(sdk *v4.LookerSDK) {
```

Comment:
Did you mean for some golang changes to sneak into this Kotlin PR?

Reply:
I'm still trying to figure out how this got into this PR. I thought perhaps it was a merge from master. @panoskoug @kppk do you know what happened here?

Reply:
Since this is already merged to master, this will be a no-op. I probably clicked wrong in IntelliJ and merged the wrong branch into mine.

Reply:
Looks like that commit was already merged to master, so somehow github has gotten confused.  Should be a no-op in the end.